css/styles.css

```diff
@@ -0,0 +1,197 @@
+* {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 auto;
+  padding: 0;
+  width: 1440px;
```

Comment:
You do not need to set an explicit width like this, if you use more responsive units on your containers this would not be needed,